x-pack/plugins/rollup/server/client/elasticsearch_rollup.js

```diff
@@ -93,10 +93,13 @@ export const elasticsearchJsPlugin = (Client, config, components) => {
   rollup.stopJob = ca({
     urls: [
       {
-        fmt: '/_rollup/job/<%=id%>/_stop',
+        fmt: '/_rollup/job/<%=id%>/_stop?wait_for_completion=<%=waitForCompletion%>',
```

Comment:
params should be defined like https://github.com/elastic/elasticsearch-js-legacy/blob/master/src/lib/apis/7_x.js#L6894-L6898

Reply:
Ah good to know. I copied this from ML (https://github.com/elastic/kibana/blob/master/x-pack/plugins/ml/server/client/elasticsearch_ml.js#L92)